src/oc/web/core.cljs

```diff
@@ -115,7 +115,17 @@
       (when (or (> (- now latest-entry-point) reload-time)
                 (and (router/current-org-slug)
                      (nil? (dis/org-data))))
-        (api/get-entry-point))
+        (api/get-entry-point
```

Comment:
@thepug we can use `user-actions/get-entry-point` here, right?
we are using action creators to avoid having api calls and similar things around the code.
WYT?